app/services/usps_in_person_proofing/mock/fixtures.rb

```diff
@@ -19,6 +19,10 @@ def self.request_facilities_response
         load_response_fixture('request_facilities_response.json')
       end
 
+      def self.request_EIPP_facilities_response
```

Comment:
I'd normally expect a method name to be entirely lower-case, even acronyms.

```suggestion
      def self.request_eipp_facilities_response
```

Reply:
My comment was marked as resolved, but it still appears to be uppercase in the code.

Reply:
I reacted with a thumbs up because I agreed with the comment, then I resolved it once I had made the change locally.

If the method appeared to include uppercase letters, I believe that’s because you were viewing the code before I had pushed up my local changes.

Reply:
b88855524c00744a63ef15d4c9cf25e09241b1f2 wasn't pushed at the time of my comment, but it looks like that addresses it.

There's still one other I'd recommend lower-casing as well:

https://github.com/18F/identity-idp/blob/b88855524c00744a63ef15d4c9cf25e09241b1f2/spec/support/usps_ipp_helper.rb#L38